src/language/dafnyInstallation.ts

```diff
@@ -24,7 +24,7 @@ const ArchiveFileName = 'dafny.zip';
 const mkdirAsync = promisify(fs.mkdir);
 
 // Equivalent to a || b but without ESLint warnings
-async function ifNullOrEmpty(a: string | null, b: () => Promise<string>): Promise<string> {
+async function ifNullOrEmptyAsync(a: string | null, b: () => Promise<string>): Promise<string> {
```

Comment:
nit: In contrast to C#, the `Async` suffix is rather uncommon in JavaScript/TypeScript. Would make a few things better though. :)